### ISSUE TITLE
diode orientation

### ISSUE THREAD

**Issue by KeLaiFu** [kraakenstuff/ADSR#3]
I'm unsure how to orientate the 0603 diodes on this pcb, I've looked at the pcb in kicad and I can see theres a taper on one side of the symbol, is this the side to match with the line on the actual component?

**Comment by lgrabenhorst** [comment]
Hey @KeLaiFu, did you manage to sort this out? I am also struggling with finding the correct orientation.

**Comment by KeLaiFu** [comment]
Yeah, my assumptions were correct, the taper in the symbols within the kicad file showed the orientation. If you are unfamiliar with kicad you may need to google how to switch off individual layers to see the symbols, but its not a big deal.
